app/cni/pkg/cni/annotations.go

```diff
@@ -144,3 +164,13 @@ func mapAnnotation(annotations map[string]string, field *string, fieldName strin
 	*field = val
 	return nil
 }
+
+func assignIPv6InboundRedirectPort(allFields map[string]*string) {
+	fieldPointer := allFields["inboundPortV6"]
+	disabledAnno := *allFields["ipv6Disabled"]
```

Comment:
is it `annotation`? also it's a pointer so a possible NPE here, we should use comma-ok check

Reply:
The value was originated from an annotation from the parent level of callsites, and they are parsed from pod annotations.